WcaOnRails/app/views/regulations/history/index.html.erb

```diff
@@ -5,6 +5,7 @@
   <p>Until 2011, the Regulations were maintained by Ron van Bruchem and the WCA Board. Since then, the <%= mail_to "wrc@worldcubeassociation.org", "WCA Regulations Committee" %> is in charge of them.<br />For the 2013 release, the Regulations were split into two documents: the Regulations and the Guidelines.</p>
   <p>Here are all past and current versions of the Regulations.</p>
   <ul>
+    <li><%= link_to "2019-05-01", "./official/2019-01-05/" %> (<%= link_to "Changes", "https://github.com/thewca/wca-regulations/compare/official-2018-01-01...release-candidate-2019-04-04" %>, <%= link_to "Summarized Changes", "https://www.worldcubeassociation.org/posts/wca-regulations-may-2019" %>)</li>
```

Comment:
I think the link is slightly off here, while the release candidate ended up being the same as the official version, we'd better use the `official-2019-05-01` tag, and also include the `#files_bucket` anchor to directly link to the diff.

Reply:
```suggestion
    <li><%= link_to "2019-05-01", "./official/2019-01-05/" %> (<%= link_to "Changes", "https://github.com/thewca/wca-regulations/compare/official-2018-01-01...official-2019-05-01" %>, <%= link_to "Summarized Changes", "https://www.worldcubeassociation.org/posts/wca-regulations-may-2019" %>)</li>
```